internal/build/job/maker.go

```diff
@@ -130,14 +139,15 @@ func MakeSecretVolumes(secretRefs []v1.LocalObjectReference) []v1.Volume {
 	return volumes
 }
 
+//FIXME:ybettan: if no one else uses it we should update it to a single object instead of a list
 func MakeSecretVolumeMounts(secretRefs []v1.LocalObjectReference) []v1.VolumeMount {
 	secretVolumeMounts := make([]v1.VolumeMount, 0, len(secretRefs))
 
 	for _, secretRef := range secretRefs {
 		volMount := v1.VolumeMount{
 			Name:      volumeNameFromSecretRef(secretRef),
 			ReadOnly:  true,
-			MountPath: "/run/secrets/" + secretRef.Name,
+			MountPath: "/kaniko/.docker",
```

Comment:
Just like above, this function was designed for build secrets, not pull secrets. This breaks the [in-cluster build e2e workflow](https://github.com/qbarrand/oot-operator/blob/main/ci/module-ooto-ci-build.template.yaml#L32) which leverages build secrets.

Reply:
Fixed.